src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier color = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(color.getRandomColor().toString(), getRandomNumber());
+    }
+
+    public int getRandomNumber() {
+        Random random = new Random();
```

Comment:
please extract random to class variable, it will also make this method unnecessary